OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/AmenityMenuBuilder.java

```diff
@@ -674,6 +682,60 @@ private String getFormattedInt(String value) {
 		}
 	}
 
+	private String[] getFormattedPrefixAndText(String key, String prefix, String value) {
+		DecimalFormat df = new DecimalFormat("#.##");
+		df.setRoundingMode(RoundingMode.CEILING);
+		switch (key){
+			case "width":
+			case "height":
+				prefix = Algorithms.capitalizeFirstLetter(key);
```

Comment:
will it be translated to other langs?
use Algorithms.capitalizeFirstLetter for locale-insensitive text only

Reply:
fixed